com.github.Murmele.scram.yml

```diff
@@ -0,0 +1,45 @@
+app-id: com.github.Murmele.scram
+runtime: org.kde.Sdk
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+command: scram-gui
+rename-icon: scram
+rename-desktop-file: scram-gui.desktop
+rename-appdata-file: scram-gui.appdata.xml
+finish-args:
+- --socket=wayland
+- --socket=fallback-x11
+- --filesystem=home
+- --share=ipc
+modules:
+- name: boost
+  buildsystem: simple
+  build-commands:
+  - "./bootstrap.sh --prefix=${FLATPAK_DEST} --with-toolset=gcc --with-libraries=filesystem,system,program_options,random"
+  - "./b2  variant=release -j $FLATPAK_BUILDER_N_JOBS install"
+  sources:
+  - type: archive
+    url: https://boostorg.jfrog.io/artifactory/main/release/1.78.0/source/boost_1_78_0.tar.bz2
+    sha256: 8681f175d4bdb26c52222665793eef08490d7758529330f98d3b29dd0735bccc
+    x-checker-data:
+      type: anitya
+      project-id: 6845
+      stable-only: true
+      url-template: https://boostorg.jfrog.io/artifactory/main/release/$version/source/boost_${major}_${minor}_$patch.tar.bz2
+
+# libxml2 is already in the sdk
+- name: scram
+  buildsystem: cmake-ninja
+  config-opts:
+  - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+  - -DBUILD_GUI=ON
+  - -DWITH_TCMALLOC=OFF
+  - -DWITH_JEMALLOC=OFF
+  - -DWITH_COVERAGE=OFF
+  - -DWITH_PROFILE=OFF
+  builddir: true
+  sources:
+  - type: git
+    url: https://github.com/Murmele/scram
```

Comment:
I noticed references to the original repo at least in the About dialog. You should update the URL there to your fork.

Reply:
ok thanks I will do